lib/occson/rails/engine.rb

```diff
@@ -1,7 +1,38 @@
+require 'occson'
+
+require_relative 'application_version'
+require_relative 'environment_storer'
+require_relative 'environment_loader'
+
 module Occson
   module Rails
     class Engine < ::Rails::Engine
       isolate_namespace Occson::Rails
+
+      config.before_configuration do
+        EnvironmentLoader.new(::Rails.root, ::Rails.env).call
+
+        unless ENV.key?('OCCSON_ACCESS_TOKEN') && ENV.key?('OCCSON_PASSPHRASE')
+          puts "Oh my, occson will not load configuration. Please setup OCCSON_ACCESS_TOKEN and OCCSON_PASSPHRASE"
```

Comment:
Let's be less dramatic about it :) Everything after the first dot will be quite fine.